openy.install

```diff
@@ -1120,6 +1120,6 @@ function openy_update_8079() {
 /**
  * Enable openy_block_branch_contacts_info module.
  */
-function openy_update_8080() {
+function openy_update_8081() {
```

Comment:
weird renaming. Why?
It could case double hook run for those who already on 8080

Reply:
Fixed